database.txt

```diff
@@ -0,0 +1 @@
+{"abe@hajemail.com": {"name": "abed", "birth_date": "24/3/2002", "email": "abe@hajemail.com", "password": "123"}, "ahma@hotmail.com": {"name": "ahmad", "birth_date": "2/5/2000", "email": "ahma@hotmail.com", "password": "aoh123456"}, "ah@.vom": {"name": "ahmad", "birth_date": "35/2008", "email": "ah@.vom", "password": "123"}, "tot@hotmail.com": {"name": "tasneem", "birth_date": "8/3/2004", "email": "tot@hotmail.com", "password": "tot123"}, "ramos.asal.com": {"name": "ramos", "birth_date": "1/5/1959", "email": "ramos.asal.com", "password": "123"}, "tripple@gmail.com": {"name": "tripleh", "birth_date": "8/25/1996", "email": "tripple@gmail.com", "password": "aoh123456"}, "ran_wwe.com": {"name": "randy orten", "birth_date": "6/8/1979", "email": "ran_wwe.com", "password": "aoh123456"}, "rami.alhaj@hotmail.com": {"name": "rami", "birth_date": "2/5/2008", "email": "rami.alhaj@hotmail.com", "password": "aoh123456"}, "omrami.alhaj@hotmail.com": {"name": "omrami", "birth_date": "2/5/2000", "email": "omrami.alhaj@hotmail.com", "password": "aoh123456"}, "tamer.bdran@hotmail.com": {"name": "tamer", "birth_date": "2/5/2000", "email": "tamer.bdran@hotmail.com", "password": "aoj123456"}, "ahmad@gmail.com": {"name": "ahmad", "birth_date": "2/5/2000", "email": "ahmad@gmail.com", "password": -1270871677306505514}, "hamada@gmail.com": {"name": "ahmad", "birth_date": "2/5/2000", "email": "hamada@gmail.com", "password": 2153163380379630782}}
```

Comment:
This file and all the other pyc files should not be included in the PR, make sure to remove them and add them to '.gitignore'

Reply:
done